{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"TimeLimitExceeded"}],"languages":["swift"],"platforms":["iOS"]}]}],"schemaVersion":{"major":0,"minor":2,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/ioginfrastructure\/iogretrymanager\/retryresult\/timelimitexceeded"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/IoGInfrastructure\/documentation\/IoGInfrastructure\/IoGRetryManager\/RetryResult\/TimeLimitExceeded","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"The retry attempt ended when it failed to succeed after the designated length of time"}],"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"TimeLimitExceeded"}],"title":"IoGRetryManager.RetryResult.TimeLimitExceeded","roleHeading":"Case","role":"symbol","symbolKind":"case","externalID":"s:17IoGInfrastructure0A13GRetryManagerC11RetryResultO17TimeLimitExceededyA2EmF","modules":[{"name":"IoGInfrastructure"}]},"hierarchy":{"paths":[["doc:\/\/IoGInfrastructure\/documentation\/IoGInfrastructure","doc:\/\/IoGInfrastructure\/documentation\/IoGInfrastructure\/IoGRetryManager","doc:\/\/IoGInfrastructure\/documentation\/IoGInfrastructure\/IoGRetryManager\/RetryResult"]]},"references":{"doc://IoGInfrastructure/documentation/IoGInfrastructure/IoGRetryManager/RetryResult/TimeLimitExceeded":{"role":"symbol","title":"IoGRetryManager.RetryResult.TimeLimitExceeded","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"TimeLimitExceeded"}],"abstract":[{"type":"text","text":"The retry attempt ended when it failed to succeed after the designated length of time"}],"identifier":"doc:\/\/IoGInfrastructure\/documentation\/IoGInfrastructure\/IoGRetryManager\/RetryResult\/TimeLimitExceeded","kind":"symbol","type":"topic","url":"\/documentation\/ioginfrastructure\/iogretrymanager\/retryresult\/timelimitexceeded"},"doc://IoGInfrastructure/documentation/IoGInfrastructure/IoGRetryManager":{"role":"symbol","title":"IoGRetryManager","fragments":[{"kind":"keyword","text":"class"},{"kind":"text","text":" "},{"kind":"identifier","text":"IoGRetryManager"}],"abstract":[{"type":"text","text":"Singleton class that manages attempts to execute sections of code that previously failed to complete"}],"identifier":"doc:\/\/IoGInfrastructure\/documentation\/IoGInfrastructure\/IoGRetryManager","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"IoGRetryManager"}],"url":"\/documentation\/ioginfrastructure\/iogretrymanager"},"doc://IoGInfrastructure/documentation/IoGInfrastructure/IoGRetryManager/RetryResult":{"role":"symbol","title":"IoGRetryManager.RetryResult","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"RetryResult"}],"abstract":[{"type":"text","text":"The disposition of the retry request after it has completed"}],"identifier":"doc:\/\/IoGInfrastructure\/documentation\/IoGInfrastructure\/IoGRetryManager\/RetryResult","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"RetryResult"}],"url":"\/documentation\/ioginfrastructure\/iogretrymanager\/retryresult"},"doc://IoGInfrastructure/documentation/IoGInfrastructure":{"role":"collection","title":"IoGInfrastructure","abstract":[],"identifier":"doc:\/\/IoGInfrastructure\/documentation\/IoGInfrastructure","kind":"symbol","type":"topic","url":"\/documentation\/ioginfrastructure"}}}